share/availability/test/testing.go

```diff
@@ -55,29 +54,27 @@ func (n *Node) ClearStorage() {
 	}
 }
 
-type DagNet struct {
+type TestDagNet struct {
 	ctx   context.Context
 	T     *testing.T
 	net   mocknet.Mocknet
-	nodes []*Node
+	nodes []*TestNode
 }
 
 // NewTestDAGNet creates a new testing swarm utility to spawn different nodes
 // and test how they interact and/or exchange data.
-func NewTestDAGNet(ctx context.Context, t *testing.T) *DagNet {
-	return &DagNet{
+func NewTestDAGNet(ctx context.Context, t *testing.T) *TestDagNet {
+	return &TestDagNet{
 		ctx: ctx,
 		T:   t,
 		net: mocknet.New(),
 	}
 }
 
-// Node create a plain network node that can serve and request data.
-func (dn *DagNet) Node() *Node {
+// NewTestNodeWithBlockstore creates a new plain TestNode with the given blockstore that can serve and request data.
```

Comment:
line wrap here and a few other places in this file.

Reply:
was already `gq`'ed